sample/src/main/java/me/zhanghai/android/materialprogressbar/sample/DeterminateCircularExamplesActivity.java

```diff
@@ -0,0 +1,138 @@
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.determinate_circular_activity);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+
+        setupDeterminateCircleProgressBars();
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case android.R.id.home:
+                AppUtils.navigateUp(this);
+                return true;
+            default:
+                return super.onOptionsItemSelected(item);
+        }
+    }
+
+    private void setupDeterminateCircleProgressBars() {
+        final MaterialProgressBar dcDefault = (MaterialProgressBar) findViewById(R.id.dc_default);
+        final MaterialProgressBar dcSecondary = (MaterialProgressBar) findViewById(R.id.dc_secondary);
+        final MaterialProgressBar dcSecondaryBackground = (MaterialProgressBar) findViewById(R.id.dc_secondary_background);
+
+        final MaterialProgressBar dcTinted = (MaterialProgressBar) findViewById(R.id.dc_tinted);
+        final MaterialProgressBar dcTintedSecondary = (MaterialProgressBar) findViewById(R.id.dc_tinted_secondary);
+        final MaterialProgressBar dcTintedSecondaryBackground = (MaterialProgressBar) findViewById(R.id.dc_tinted_secondary_background);
+
+        final MaterialProgressBar dcmDefault = (MaterialProgressBar) findViewById(R.id.dc_moving_default);
+        final MaterialProgressBar dcmSecondary = (MaterialProgressBar) findViewById(R.id.dc_moving_secondary);
+        final MaterialProgressBar dcmSecondaryBackground = (MaterialProgressBar) findViewById(R.id.dc_moving_secondary_background);
+
+        final MaterialProgressBar dcmTinted = (MaterialProgressBar) findViewById(R.id.dc_moving_tinted);
+        final MaterialProgressBar dcmTintedSecondary = (MaterialProgressBar) findViewById(R.id.dc_moving_tinted_secondary);
+        final MaterialProgressBar dcmTintedSecondaryBackground = (MaterialProgressBar) findViewById(R.id.dc_moving_tinted_secondary_background);
+
+        final List<MaterialProgressBar> all = Arrays.asList(
+                dcDefault,
+                dcSecondary,
+                dcSecondaryBackground,
+                dcTinted,
+                dcTintedSecondary,
+                dcTintedSecondaryBackground,
+
+                dcmDefault,
+                dcmSecondary,
+                dcmSecondaryBackground,
+                dcmTinted,
+                dcmTintedSecondary,
+                dcmTintedSecondaryBackground
+        );
+
+        final List<MaterialProgressBar> withSecondary = Arrays.asList(
+                dcSecondary,
+                dcSecondaryBackground,
+                dcTintedSecondary,
+                dcTintedSecondaryBackground,
+
+                dcmSecondary,
+                dcmSecondaryBackground,
+                dcmTintedSecondary,
+                dcmTintedSecondaryBackground
+        );
+
+        new Thread() {
```

Comment:
Really shouldn't kick off thread for animation in Android; In most cases one should use `ObjectAnimator`, and it is much more smoother. I'll refactor this later.